src/main/java/io/fusionauth/domain/oauth2/UserState.java

```diff
@@ -47,10 +47,6 @@ public static UserState fromStatus(int status) {
     throw new IllegalArgumentException("Invalid status code for UserState [" + status + "]");
   }
 
-  /**
-   * This method only applies to legacy user identities before 1.99.9 and should no longer be used.
-   */
-  @Deprecated
```

Comment:
You need to merge in feature branch to not erase this. My `userState` changes must have been merged after you started this feature.